build/docker/entrypoint.sh

```diff
@@ -43,10 +43,12 @@ initialize_log_dir() {
 }
 
 copy_provided_plugins() {
-  if [ -d "/opt/plugins" ]; then
-    echo "Copying user-provided plugins"
-    cp -nR /opt/plugins/* ${OPENFIRE_DIR}/plugins/
-  fi
+  for file in /opt/plugins/*.jar; do # Workaround for https://github.com/koalaman/shellcheck/wiki/SC2144 that doesn't involve redirecting terminal output
+    if [ -f "$file" ]; then
+      echo "Copying user-provided plugins"
```

Comment:
Can we add the $file in the echo message?  Otherwise, that echo comes for each plugin within the loop?